app/controllers/api/v2/accounts_controller.rb

```diff
@@ -17,8 +17,12 @@ def create
     @user, @account = AccountBuilder.new(
       email: account_params[:email],
       user_password: account_params[:password],
+      locale: account_params[:locale],
```

Comment:
Is there an associated change in the account builder?

Reply:
Yep, it handles that already

https://github.com/chatwoot/chatwoot/blob/eb379e184968f77546e9a30def93e56c31d38a0c/app/builders/account_builder.rb#L5

https://github.com/chatwoot/chatwoot/blob/eb379e184968f77546e9a30def93e56c31d38a0c/app/builders/account_builder.rb#L51-L54